data/data/bootstrap/files/etc/systemd/system/node-image-pull.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Node Image Pull
+Requires=network.target NetworkManager.service
+After=network.target
+
+# for easier debugging
+Requires=sshd.service getty.target systemd-user-sessions.service
+After=sshd.service getty.target systemd-user-sessions.service
+
+[Service]
+# we use Type=notify so that systemd keeps retrying until we succeed. With
```

Comment:
A useful trick...do you have a link for that?

Reply:
Possibly this came from https://github.com/systemd/systemd/issues/1312#issuecomment-279172631
FWIW I attempted to use this in https://github.com/openshift/installer/pull/8911 and evidently it didn't work.
Consensus on the thread is that only https://github.com/systemd/systemd/issues/1312#issuecomment-228874771 works until we get systemd 254 and `RestartMode=direct`.

Reply:
OK yes indeed, that doesn't work. I think this appeared to work when I was testing this but was actually for a different reason.

> Consensus on the thread is that only https://github.com/systemd/systemd/issues/1312#issuecomment-228874771 works until we get systemd 254 and `RestartMode=direct`.

Hmm, that requires having service B (the dependent service) also have `Restart=`. I think a cleaner workaround until `RestartMode=direct` is to just have in service A, `ExecStartPost=-systemctl start --no-block b.service`. Obviously in that case, you wouldn't have an `[Install]` section or a `Requires=`.